### PR TITLE
B2B-1112 Team Feedback

### PR DESCRIPTION
**Pull request by front-eric**

### CODE REVIEW

**Review by scott-collins-mesh — COMMENTED** [review]
Various TypeScript errors when I past sample code from Readme in IDE.

**Comment by vitaliimovchan-front** [comment]
Missing: 7. Let’s move [frontFinanceModule](https://github.com/FrontFin/front-b2b-link-rn/tree/main/example/frontFinanceModule) to the root directory.

**Comment by vitalii-movchan-mesh** [comment]
Missing:
1. Add types for onReceive, onError callbacks like it done for `front-b2b-link`.
4. Sample code should not have any compile issues.
7. Let’s move `frontFinanceModule` to the root directory.

File: README.md [hunk 1]
```diff
@@ -5,72 +5,75 @@
...
-   Front-Finance-rn-sdk component.
-
-3. Front-Finance-rn-sdk component is a react component which will take a url and return account details against your selected broker account for e.g. RobinHood etc...
+2. Front-Finance-rn-sdk component is a react component which will take a url and return account details against your selected broker account for e.g. RobinHood etc...
 
 ```js
-import React, {useEffect} from 'react'
-import {FrontFinance} from '@front-finance/frontfinance-rn-sdk';
-import { FrontApi } from "@front-finance/api";
+import React, { useState } from "react";
```
Comment (scottcollins-front) [inline README.md:52]: useState is not used, can remove

**Comment by balint-front** [comment]
@front-eric Hey. just had a quick call with Alex regarding locally testing and distributing the SDK. I think it would be useful to introduce a monorepo structure to this project, where we would have the root package (monorepo) that contains two child packages: the SDK itself, which is distributed. and the example app package, which is shared only by code in a public repo. 

What are your thoughts?

File: example/utility/config.js [hunk 2]
```diff
@@ -5,3 +5,28 @@ export const clientId = Config.REACT_APP_CLIENT_ID
...
+      {
+        NetworkId: "e3c7fdd8-b1fc-4e51-85ae-bb276e075611",
+        Symbol: "ETH",
+        Address: "0x9Bf6207f8A3f4278E0C989527015deFe10e5D7c6",
+      },
+      {
+        NetworkId: "e3c7fdd8-b1fc-4e51-85ae-bb276e075611",
+        Symbol: "USDC",
+        Address: "0x9Bf6207f8A3f4278E0C989527015deFe10e5D7c6",
+      },
```
Comment (vitaliimovchan-front) [inline example/utility/config.js:19]: this should be removed

**Review by vitalii-movchan-mesh — APPROVED** [review]

**Pull request merged**